presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/README.md

```diff
@@ -0,0 +1,148 @@
+# Presto planner and optimizer tests
+
+Testing planner and optimizer functionality generally requires generating a
+plan and then validating it against a set of expectations. The challenge is
```

Comment:
Suggest deleting "generally"
Is it a query plan that's generated? (See query plans in line 18.)